tests/model_serving/model_server/inference_service_configuration/conftest.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Generator
+
+import pytest
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.pod import Pod
+
+from tests.model_serving.model_server.inference_service_configuration.constants import (
+    ISVC_ENV_VARS,
+)
+from tests.model_serving.model_server.inference_service_configuration.utils import (
+    update_inference_service,
+)
+from utilities.infra import get_pods_by_isvc_label
+
+
+@pytest.fixture(scope="class")
+def removed_isvc_env_vars(
+    request: pytest.FixtureRequest,
+    admin_client: DynamicClient,
+    ovms_kserve_inference_service: InferenceService,
+) -> Generator[InferenceService, Any, Any]:
+    if isvc_predictor_spec_model_env := ovms_kserve_inference_service.instance.spec.predictor.model.get("env"):
```

Comment:
If there is no "env", this code would silently pass. I know you mentioned that is not a possibility, then we are using "get('env')" instead of ["env"].

Reply:
added `else` to raise if not envs